src/main/steps/upload-documents/content.test.ts

```diff
@@ -89,7 +89,7 @@ const welshContent = () => ({
   errors: {
     documentUpload: {
       noInput: 'You cannot continue without providing additional information or a document - welsh',
-      fileSize: 'File size exceeds the maximum permitted value. Please upload a file that is less than 500MB - welsh',
+      fileSize: 'File size exceeds the maximum permitted value. Please upload a file that is less than 100MB - welsh',
```

Comment:
Shouldn't this be "Mae’r ffeil yn fwy na’r maint mwyaf a ganiateir. Llwythwch ffeil sy’n llai na 100MB"?